site/content/en/v1.0.1/tasks/traffic/client-traffic-policy.md

```diff
@@ -412,7 +412,7 @@ Handling connection for 8888
 
```

Comment:
Is this feature available in 1.0.1 ? I only see the commit in latest: https://github.com/envoyproxy/gateway/commit/6941624e1fbe2e3c2f9d8ef47b71862cc4a53749

Reply:
removed, thanks